proto/ibc/lightclients/wasm/v1/tx.proto

```diff
@@ -19,12 +18,12 @@ message MsgStoreCode {
   option (cosmos.msg.v1.signer) = "signer";
 
   string signer = 1;
-  // WASMByteCode can be raw or gzip compressed
-  bytes wasm_byte_code = 2 [(gogoproto.customname) = "WASMByteCode"];
+  // wasm byte code of light client contract. It can be raw or gzip compressed
+  bytes wasm_byte_code = 2;
```

Comment:
didn't you want to keep the gogoproto custom name? 

https://github.com/CosmWasm/wasmd/blob/main/proto/cosmwasm/wasm/v1/tx.proto#L83

Reply:
Oh, I thought [you preferred to have the field name as `WasmByteCode`](https://github.com/cosmos/ibc-go/pull/4091#discussion_r1264379324).